Gemfile

```diff
@@ -49,5 +93,10 @@ group :development do
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
 
+group :production do
+  # Use postgresql as the database for Active Record
+  gem 'pg', '~> 0.21.0'
```

Comment:
On production, I used Heroku for deploy and Heroku require `gem pg`.
Normally, I used postgresql as default for development and Test environment.